chinemelu/fast-food-fast-React
# #162125609 Setup webpack

#### What does this PR do?
Have Webpack working on project

#### Description of Task to be completed?
* Install necessary dependencies
* Setup webpack

#### How should this be manually tested?
After cloning the repo, cd into it, run `npm install` and run `npm run start`. Go to browser and type `http://localhost:8080/`. Go to the developer tools console and there should be `Hey guys and ladies!!` written in the console

#### Any background context you want to provide?
NA

#### What are the relevant pivotal tracker stories?
[#162125609](https://www.pivotaltracker.com/story/show/162125609)

#### Screenshots (if appropriate)
NA 

#### Questions:
NA
